content/en/blog/_posts/2022-05-13-grpc-probes-in-beta.md

```diff
@@ -28,11 +28,11 @@ Ahmet Alp Balkan described how you can do that — a mechanism that still works
 
 There is a commonly used tool to enable this that was [created](https://github.com/grpc-ecosystem/grpc-health-probe/commit/2df4478982e95c9a57d5fe3f555667f4365c025d)
 on August 21, 2018, and with
-the first release at [Sep 19, 2018](https://github.com/grpc-ecosystem/grpc-health-probe/releases/tag/v0.1.0-alpha.1).
+the first release at [Sep 20, 2018](https://github.com/grpc-ecosystem/grpc-health-probe/releases/tag/v0.1.0-alpha.1).
```

Comment:
The original date is correct for a UTC timezone.

Reply:
ok, reverted this change.